### PR TITLE
Sixgill darkfeed feed tags

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/22895

## Description
Added feed tags param

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py [hunk 1]
```diff
@@ -531,3 +531,24 @@ def test_get_indicators_command(mocker):
     submitted_indicators = 0
 
     assert output[2] == expected_ioc_output
+
+
+def test_feed_tags(mocker):
```
Comment (bakatzir) [inline Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py:536]: please add given/when/then

File: Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py [hunk 2]
```diff
@@ -531,3 +531,24 @@ def test_get_indicators_command(mocker):
     submitted_indicators = 0
 
     assert output[2] == expected_ioc_output
+
+
```
Comment (bakatzir) [inline Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py:535]: please consider adding 1 more test case where there is a tag given by the user in the params, and one coming from the feed as an IOC `label`, as the code implies.

Comment (orhovy) [reply Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py:535]: That is the current test case, as the result already has [ 'compromised', 'shell', 'webshell'] in the "label" and ['tag1','tag2'] are being added to it.

Comment (bakatzir) [reply Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed_test.py:535]: I see it in the data, can you add 1 more assert for it in `test_feed_tags`?
So we can check that the tags coming from. the feed are also present. Or am I missing your explanation?

**Pull request closed without merging**